### PR TITLE
Changed branding to Solar2D

### PR DESCRIPTION
**Pull request by sekodev**
This is for comments, error messages and warnings shown in code and simulator.

### CODE REVIEW

File: platform/mac/Credits.rtfd/TXT.rtf [hunk 1]
```diff
@@ -4,5 +4,5 @@
 \margl1440\margr1440\vieww12600\viewh10200\viewkind0
 \pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\qc\partightenfactor0
 
-\f0\b\fs24 \cf0 Copyright \'a9 2020- Solar2D\
+\f0\b\fs24 \cf0 Copyright \'a9 2020- Solar2D.\
```
Comment (Destiny-2048) [inline platform/mac/Credits.rtfd/TXT.rtf:7]: Hi :) It seems to me that the dash after "2020" should be removed.

Comment (Shchvova) [reply platform/mac/Credits.rtfd/TXT.rtf]: Fixed

File: platform/mac/Info.plist [hunk 2]
```diff
@@ -76,7 +76,7 @@
 	<key>NSHighResolutionCapable</key>
 	<true/>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2009-2020 Corona Labs Inc. All rights reserved.</string>
+	<string>Copyright © 2020- Solar2D. All rights reserved.</string>
```
Comment (Destiny-2048) [inline platform/mac/Info.plist:79]: And here

File: platform/mac/AppDelegate.mm [hunk 3]
```diff
@@ -1107,7 +1107,7 @@ - (void) loadExtensionMenu
     NSMenu *appMenu = [[NSApplication sharedApplication] mainMenu];
     NSMenuItem *windowMenuItem = [appMenu itemWithTitle:kWindowMenuItemName];
     NSMenu *windowMenu = [windowMenuItem submenu];
-    long welcomeItemIdx = [windowMenu indexOfItemWithTitle:@"Welcome to Corona"];
+    long welcomeItemIdx = [windowMenu indexOfItemWithTitle:@"Welcome to Solar2D"];
```
Comment (Shchvova) [inline platform/mac/AppDelegate.mm:1110]: This would cause an issue, since it is trying to fetch menu by the name. Will rename menu as well.

Comment (Shchvova) [reply platform/mac/AppDelegate.mm:1110]: Fixed welcome window title and menu item to be Solar2D

**Comment by Shchvova** [comment]
Thanks @sekodev and @Destiny-2048 a lot!

**Pull request merged**